### PR TITLE
Phase2-hgx260 Bug fix for HGCal geometry relevant for V14

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Partial bug fix for HGCal geometry relevant for V14

#### PR validation:

Tested using runTheMatrix.py workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+upgrade

File: Geometry/HGCalGeometry/test/HGCalWaferInFileTest.cc [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "Geometry/HGCalGeometry/interface/HGCalGeometry.h"
```
Comment (cvuosalo) [inline Geometry/HGCalGeometry/test/HGCalWaferInFileTest.cc:37]: Looks like `FWCore/ParameterSet` and `Geometry/HGCalGeometry` should be added to `Geometry/HGCalCommonData/test/BuildFile.xml`

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by silviodonato** [comment]
urgent

**Comment by silviodonato** [comment]
@cms-sw/geometry-l2 (@cvuosalo ) do you have further comments?

**Comment by kpedro88** [comment]
+upgrade

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**